cpp/ql/src/experimental/Security/CWE/CWE-561/FindIncorrectlyUsedSwitch.qhelp

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>Finding places the dangerous use of a switch.</p>
```

Comment:
This overview doesn't really tell me why I should care about the alerts that this query finds. What does a dangerous switch look like? And _why_ are they dangerous?

Reply:
in my opinion the danger of these situations is in violation of flow control (cwe-691) when the developer does not handle all the choices or in dead code (cwe-561)

Reply:
That sounds great. Could you write replace the "Finding places the dangerous use of a switch." paragraph with a sentence like that?

Reply:
would such a text work?  
```
search for places of incomplete correspondence of condition values and selection values. This can lead to a flow control violation (cwe-691) where the developer does not handle all choices, or dead code (cwe-561)
```

Reply:
I like it! I've taken inspiration from your text in the following:
```
A mismatch between conditionals and <code>switch</code> cases can lead to control-flow violations (CWE-691) where the developer either does not handle all combinations of conditions or unintentionally created dead code (CWE-561).
```
I think that one is phrased in a more standard way. Do you think that still captures your intended meaning?

Reply:
Thank you very much